static/js/containers/TermsOfServicePage.js

```diff
@@ -59,8 +59,8 @@ class TermsOfServicePage extends React.Component {
                 <ol>
                   <li>
                     Your use of this Site is entirely voluntary.  This Site is designed to assist and facilitate your
-                    successful attainment of the MITx MicroMaster’s credential, but use of this Site is not required
-                    in order to participate in the MITx MicroMaster’s program. These terms and conditions govern
+                    successful attainment of the MITx MicroMaster's credential, but use of this Site is not required
```

Comment:
remove single quote. I think MicroMasters is used more as a adjective here than a possessive